### DISCUSSION TITLE
Find your scope tools @ INSPIRE knowledge base tools

### DISCUSSION THREAD

**Discussion by MartinTuchyna** [INSPIRE-MIF/helpdesk#190] [Q&A] [answered]
Dear INSPIRE Team,

could you please extend the [Tools section of the INSPIRE knowledge base](https://knowledge-base.inspire.ec.europa.eu/tools_en) with the icon/link to [Find your scope set of tools](https://inspire-regadmin.jrc.ec.europa.eu/dataspecification/FindYourScope.action) as on [old version of INSPIRE website](https://wayback.archive-it.org/12090/20230902103304/https://inspire-regadmin.jrc.ec.europa.eu/dataspecification/FindYourScope.action)?

![obrázok](https://github.com/INSPIRE-MIF/helpdesk/assets/6195936/bb252227-fb63-4b04-bc95-b7ae6a1fda57)

![obrázok](https://github.com/INSPIRE-MIF/helpdesk/assets/6195936/c6a57467-d6c5-4d67-b5a2-9c6f3609dc1f)

Thanks in advance,

Martin

**Comment by fabiovinci** [comment, marked as answer]
Dear @MartinTuchyna,

the "Find Your Scope" tool was decommissioned as all information was encoded within the tool's code and was therefore out of date.
